deploy/metrics-apiservice.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: apiregistration.k8s.io/v1beta1
+kind: APIService
+metadata:
+  name: v1alpha1.metrics
+spec:
+  insecureSkipTLSVerify: true
+  group: metrics
```

Comment:
Yeah, I followed what is in metrics repos. I'm not convinced that we want to change this though. Do we plan to change `autoscaling` to `autoscaling.k8s.io`? I think `metrics` api group is a first class api group in Kubernetes (whatever it means) similarly to `apps` or `autoscaling`. Only due to performance reasons it's implemented through another apiserver.